src/components/PauseLiquidations.tsx

```diff
@@ -0,0 +1,106 @@
+import clsx from 'clsx';
+import { motion } from 'framer-motion';
+import { WalletContext } from 'lib/wallet';
+import { useContext, useState } from 'react';
+
+interface Props {
+  charterOfferId: number;
+}
+
+const invitationDescriptions = {
+  newBid: 'new bid',
+};
+
+export default function PauseLiquidations(props: Props) {
+  const walletUtils = useContext(WalletContext);
+  // XXX read the initial state from rpc?
```

Comment:
good callout. does the UI make clear to the user they're not seeing the current state? I don't see that in this PR

Reply:
It's indicated by the text "Current filter not displayed" 
![image](https://user-images.githubusercontent.com/8848650/232960761-e132a488-07e9-4b05-a2bb-2ba97f1c3ca5.png)